coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -57,11 +58,23 @@ def apply(self, result, original_file_dict, file_diff_dict, language: str,
         filename = source_range.file
 
         ignore_diff = Diff(original_file_dict[filename])
-        ignore_diff.change_line(
-            source_range.start.line,
-            original_file_dict[filename][source_range.start.line-1],
-            original_file_dict[filename][source_range.start.line-1].rstrip() +
-            '  ' + ignore_comment)
+
+        original_line = str(original_file_dict[filename][
+                        source_range.start.line-1].rstrip())
+
+        # Add the new ignore using 'and' with the previous ignore.
+        if re.compile(r'(Ignore\s)(.+Bear)').search(original_line):
```

Comment:
you can use `re.search` directly instead of compiling it before ;) That's only useful if you keep the returned expression-object around for performance reasons ;)